### ISSUE TITLE
AVT 3 - React Search Component, the purpose of the X button not announced by VO

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#3334]
## Environment
macOS Mojave version 10.14.5
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Carbon v10 - React
Voice Over

## Detailed Description
Start VoiceOver
Navigate to the search input field
Type text in the field and the X button displays
Tab to the X button
VO announces, "Button. To click this button press Control-Option-Space"

The button should have a label that describes what the button does. (i.e. clear search)
